packages/airplane/internal/api/client.ts

```diff
@@ -87,10 +87,10 @@ export class Client {
     }>("/v0/sessions/get", { id: sessionID });
   }
 
-  async createPrompt(params: ParamSchema[]): Promise<string> {
+  async createPrompt(prompt: PromptConfig): Promise<string> {
     const resp = await this.fetcher.post<{
       id: string;
-    }>("/v0/prompts/create", { schema: { parameters: params } });
+    }>("/v0/prompts/create", { ...prompt });
```

Comment:
You probably don't need the shallow clone.

```suggestion
    }>("/v0/prompts/create", prompt);
```